domains/measurements/Measurements.API/Tests/Measurements/Helpers/TimeSeriesHelperTests.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Xunit;
+using API.Measurements.Helpers;
+using FluentAssertions;
+
+namespace Tests.Measurements.Helpers;
+
+public class TimeSeriesHelperTests
+{
+    [Theory]
+    [InlineData("2022-01-01T12:34:56.789+00:00", "2022-01-01T12:00:00.000+00:00")]
+    [InlineData("2022-01-01T12:34:56.789+02:00", "2022-01-01T10:00:00.000+00:00")]
+    public void ZeroedHour_GivenInput_ZeroOffsetToUtc(string inputTimestamp, string expectedTimestamp)
+    {
+        var actual = DateTimeOffset.Parse(inputTimestamp).ToUnixTimeSeconds().ZeroedHour();
+
+        Assert.Equal(DateTimeOffset.Parse(expectedTimestamp), actual);
+        Assert.Equal(TimeSpan.Zero, actual.Offset);
+    }
+
+    [Theory]
```

Comment:
Why make this a theory when only one input is being used?

Reply:
Cause then it is easy to extend in case of other cases